packages/core/examples/numericStepperExample.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+
+import { NumericStepper } from "../src";
+import { Intent } from "../src/common/intent";
+import BaseExample from "./common/baseExample";
+
+export class NumericStepperExample extends BaseExample<{}> {
+
+    protected renderExample() {
+        return (
+            <div className="pt-numeric-stepper-example">
```

Comment:
Still need to write real docs and implement controlled examples, but these examples are helpful for now.